src/Faker/Provider/Uuid.php

```diff
@@ -54,4 +54,18 @@ public static function uuid()
 
         return $uuid;
     }
+
+    /**
+     * Generation code from https://gist.github.com/JanTvrdik/655b31e78e998bb468c7f8f8cfe4b3b6
+     */
+    public static function uuid4() : string
+    {
+        $uuidBin = random_bytes(18);
```

Comment:
not good. You have to use the `mt_` random generation functions to allow seeding, or the generators from the `Base` object.